dm/proto/tracer_syncer.proto

```diff
@@ -0,0 +1,59 @@
+syntax = "proto3";
+
+package pb;
+
+import "tracer_type.proto";
+
+
+message BaseEvent {
+    string filename = 1;
+    int32 line = 2;
+    int64 tso = 3;
+    string traceID = 4;
+    string groupID = 5;
+    TraceType type = 6;
+}
+
+message MySQLPosition {
+    string name = 1;
+    uint32 pos = 2;
+}
+
+message SyncerState {
+    bool safeMode = 1;
+    bool tryReSync = 2;
+    MySQLPosition lastPos = 3;
+}
+
+message ExecDDLInfo {
+    string lockID = 1;
+    bool exec = 2;
+}
+
+message SyncerBinlogEvent {
+    BaseEvent base = 1;
+    SyncerState state = 2;
+    int32 eventType = 3;
+    int32 opType = 4;
```

Comment:
any `enum` for `eventType` and `opType`?

Reply:
eventType and opType are introduced from `dm/syncer` pacakge, it is a little weird